crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1,3 +1,5 @@
+//! Types for storing and manipulating ECS data.
```

Comment:
I personally don't find this line particularly useful / descriptive. I'd honestly prefer omitting it, given that World now has solid docs.